libs/ai-endpoints/langchain_nvidia_ai_endpoints/chat_models.py

```diff
@@ -220,14 +220,14 @@ def __init__(self, **kwargs: Any):
         super().__init__(**kwargs)
         self._client = _NVIDIAClient(
             base_url=self.base_url,
-            model=self.model,
-            default_model=self._default_model,
+            model_name=self.model,
+            default_hosted_model_name=self._default_model,
```

Comment:
This might break the code in setting default model for hosted endpoints

```
    @root_validator(pre=True)
    def _preprocess_args(cls, values: Any) -> Any:
        values["client"] = NVEModel(**values)

        if "base_url" in values:
            values["is_hosted"] = urlparse(values["base_url"]).netloc in [
                "integrate.api.nvidia.com",
                "ai.api.nvidia.com",
            ]

        # set default model for hosted endpoint
        if values["is_hosted"] and not values["model"]:
            values["model"] = values["default_model"]

        return values
```
can you update the default_model to default_hosted_model_name?